### DISCUSSION TITLE
Could we get a simple nicegui.ui.div that represents a html div with no styling applied (including no quasar styling)?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [zauberzeug/nicegui#737] [Ideas / Feature Requests]
Sometimes, I find that the default styling that comes with elements like `ui.column()` to be more problematic than useful when applying styling of my own. Up until now Ive been using a fair lot of:
```py
with ui.column() as column:
    column._classes.clear()
    column.style(add="my styling")
```

**Comment by falkoschindler** [comment]
Good point, @ItsCubeTime!
Short answer: You can use `ui.element('div')`.
Long answer: Please see https://github.com/zauberzeug/nicegui/discussions/572#discussioncomment-5371423 for my thoughts on you're proposal.

Let's close this ticket as duplicate and continue the discussion over at https://github.com/zauberzeug/nicegui/discussions/572. 🙂
